src/gemdat/jumps.py

```diff
@@ -300,28 +305,25 @@ def split(self, n_parts) -> list[Jumps]:
         ]
 
     @weak_lru_cache()
-    def jumps_counter_parts(self, n_parts) -> list[Counter]:
-        """Return [jump counters][gemdat.sites.SitesData.jumps] per part."""
-
-        return [part.jumps_counter() for part in self.split(n_parts)]
-
-    @weak_lru_cache()
-    def rates(self) -> dict[tuple[str, str], tuple[float, float]]:
+    def rates(self,
+              n_parts: int = 10) -> dict[tuple[str, str], tuple[float, float]]:
```

Comment:
Jumps.rates might be better returned as pandas DataFrame, feel free to make an issue from this

Reply:
See #249